content/docs/components-and-props.md

```diff
@@ -122,13 +121,13 @@ ReactDOM.render(
 
 [](codepen://components-and-props/composing-components)
 
-Typically, new React apps have a single `App` component at the very top. However, if you integrate React into an existing app, you might start bottom-up with a small component like `Button` and gradually work your way to the top of the view hierarchy.
+Por lo general, aplicaciones nuevas en React tienen un único componente `App` en lo mas alto. Sin embargo, si se integra React en una aplicación existente, se podría empezar de abajo hacia arriba con un pequeño componente como `Button` y poco a poco trabajar el camino a la cima de la jerarquía de la vista.
```

Comment:
I think
> ... las aplicaciones de React nuevas tienen un único componente `App` en lo mas alto.

is closer to what the original text is trying to convey.

Reply:
Well, my reasoning goes by thinking that they are not new applications from React but rather new React applications, which are not the same thing.

Reply:
Or maybe it can be `las aplicaciones recién creadas` even if it's not the exact translation. I thing that its meaning is more or less the same. Or as far as I understand it.

Reply:
The docs are referring to new React applications that don't have any existing code, it doesn't say anything about how recent they are.